contracts/clients/package.json

```diff
@@ -22,7 +22,7 @@
   },
   "dependencies": {
     "@thehubbleproject/bls": "^0.5.1",
-    "ethers": "5.5.4",
+    "ethers": "^5.7.2",
```

Comment:
Maybe don't have to do this as part of this PR but when we update the bls-wallet-client version in the aggregator and the extension, we might want to update ethers there too.